src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs

```diff
@@ -19,12 +21,6 @@ public class ThreadsHandler : RequestHandler
         [RavenAction("/admin/debug/threads/stack-trace", "GET", AuthorizationStatus.Operator)]
         public Task StackTrace()
         {
-            if (PlatformDetails.RunningOnPosix)
```

Comment:
I think that we need to keep it, but for MacOS?

Reply:
Yes